products/ruby.md

```diff
@@ -24,26 +24,26 @@ identifiers:
 releases:
 -   releaseCycle: "3.2"
     eol: 2026-03-31
-    latest: "3.2.1"
-    latestReleaseDate: 2023-02-08
+    latest: "3.2.2"
+    latestReleaseDate: 2023-03-30
```

Comment:
We picked rubychangelogs (unofficial) because it was more readable and gave a better better changelog. 

https://github.com/kowal/ruby-changelog seems currently unmaintained (Will file an issue, so we should switch for now).